src/Okta.Sdk.UnitTests/SessionsScenarios.cs

```diff
@@ -0,0 +1,138 @@
+                CancellationToken.None);
+        }
+
+        [Fact]
+        public async Task DelegateAValidGetToRequestExecutor()
+        {
+            var mockRequestExecutor = Substitute.For<IRequestExecutor>();
+            mockRequestExecutor
+                .GetAsync(Arg.Any<string>(), Arg.Any<IEnumerable<KeyValuePair<string, string>>>(), Arg.Any<CancellationToken>())
+                .Returns(new HttpResponse<string>() { StatusCode = 200 });
+
+            var dataStore = new DefaultDataStore(mockRequestExecutor, new DefaultSerializer(), new ResourceFactory(null, null), NullLogger.Instance);
+
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Sessions.GetSessionAsync("foo");
+
+            await mockRequestExecutor.Received().GetAsync(
+                "/api/v1/sessions/foo",
+                Arg.Any<IEnumerable<KeyValuePair<string, string>>>(),
+                CancellationToken.None);
+        }
+
+        [Fact]
+        public async Task DelegateAValidPostToRequestExecutorWhenRefreshingSession()
+        {
+            var mockRequestExecutor = Substitute.For<IRequestExecutor>();
+            mockRequestExecutor
+            .PostAsync(Arg.Any<string>(), Arg.Any<IEnumerable<KeyValuePair<string, string>>>(), Arg.Any<string>(), Arg.Any<CancellationToken>())
+            .Returns(new HttpResponse<string>() { StatusCode = 200 });
+
+            var dataStore = new DefaultDataStore(mockRequestExecutor, new DefaultSerializer(), new ResourceFactory(null, null), NullLogger.Instance);
+
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Sessions.RefreshSessionAsync("foo");
+
+            await mockRequestExecutor.Received().PostAsync(
+                "/api/v1/sessions/foo/lifecycle/refresh",
+                Arg.Any<IEnumerable<KeyValuePair<string, string>>>(),
+                null,
+                CancellationToken.None);
+        }
+
+        [Fact]
+        public async Task DelegateAValidDeleteToRequestExecutorWhenEndingSession()
+        {
+            var mockRequestExecutor = Substitute.For<IRequestExecutor>();
+            mockRequestExecutor
+               .DeleteAsync(Arg.Any<string>(), Arg.Any<IEnumerable<KeyValuePair<string, string>>>(), Arg.Any<CancellationToken>())
+               .Returns(new HttpResponse<string>() { StatusCode = 200 });
+
+            var dataStore = new DefaultDataStore(mockRequestExecutor, new DefaultSerializer(), new ResourceFactory(null, null), NullLogger.Instance);
+
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Sessions.EndSessionAsync("foo");
+
+            await mockRequestExecutor.Received().DeleteAsync(
+                "/api/v1/sessions/foo",
+                Arg.Any<IEnumerable<KeyValuePair<string, string>>>(),
+                CancellationToken.None);
+        }
+
+        [Fact]
+        public async Task ThrowApiExceptionFor401()
```

Comment:
We can remove this test, it should already be covered by this one: https://github.com/okta/okta-sdk-dotnet/blob/master/src/Okta.Sdk.UnitTests/OktaClientShould.cs#L47-L74